server/src/main/java/org/eclipse/openvsx/json/UserPublishInfoJson.java

```diff
@@ -0,0 +1,47 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+package org.eclipse.openvsx.json;
+
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;;
+
+@ApiModel(
+    value = "UserPublishInfo",
+    description = "User publish info"
+)
```

Comment:
We don't list the AdminAPI endpoints in the public API documentation, so it's not necessary to add `@ApiModel` and `@ApiModelProperty` annotations here.